python/lsst/analysis/tools/analysisMetrics/analysisMetrics.py

```diff
@@ -291,3 +296,49 @@ def setDefaults(self):
             "stdevSky": "nJy",
             "sigmaMADSky": "nJy",
         }
+
+
+class StellarPhotometricRepeatabilityMetric(AnalysisMetric):
+    parameterizedBand: bool = True
+    fluxType: str = "psfFlux"
```

Comment:
Are you planning on subclassing in some way? Why have this class level variable?

Reply:
I wanted to ensure that the same fluxType is used throughout the calculation. Is there a better way to do that?